### PR TITLE
Add http executor

### PR DESCRIPTION
**Pull request by gustavosbarreto**
executor_config options:

url: url of http request
type: request method (any of https://golang.org/pkg/net/http/#pkg-constants)
body: request body encoded in base64
content_type: request content type

### CODE REVIEW

**Comment by victorcoder** [comment]
@gustavosbarreto would be nice to have this executor for the next release. Do you think you can take care of the polishing?

**Comment by gustavosbarreto** [comment]
@victorcoder What do I need to improve?

File: builtin/bins/dkron-executor-http/http.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+)
+
+type Http struct {
+}
+
+// Process method of the plugin
+func (s *Http) Execute(args *dkron.ExecuteRequest) ([]byte, error) {
+	output, _ := circbuf.NewBuffer(maxBufSize)
+
+	if args.Config["url"] == "" {
```
Comment (victorcoder) [inline builtin/bins/dkron-executor-http/http.go]: What will happen if url is not set?

File: builtin/bins/dkron-executor-http/http.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		return nil, errors.New("url is empty")
+	}
+
+	if args.Config["method"] == "" {
```
Comment (victorcoder) [inline builtin/bins/dkron-executor-http/http.go]: Same here

**Comment by ProPheT777** [comment]
do we have a chance to see it in next release ? :pray: :heart:

**Comment by redben** [comment]
Is it possible to add headers?

**Pull request closed without merging**